resources/sdk/clisdkclient/extensions/models/experimentalfeaturecommands.go

```diff
@@ -2,8 +2,9 @@ package models
 
```

Comment:
You can go ahead and do some refactoring here. `allCommands` declared in resources/sdk/clisdkclient/extensions/experimental_features/experimental_features.go should be declared here as a global variable. It should just hold AlternativeFormats at the moment.